runtime/restart/monitor/change.go

```diff
@@ -64,6 +56,16 @@ func (s *startChange) apply(ctx context.Context, client *containerd.Client) erro
 			s.logPath, s.logURI)
 	}
 
+	task, err := s.container.Task(ctx, nil)
+	if err == nil {
+		if status, err := task.Status(ctx); err == nil {
+			if status.Status != containerd.Stopped {
+				return nil
+			}
+		}
+		task.Delete(ctx)
```

Comment:
why not use `killTask(ctx, s.container)` here?

Reply:
If a task has been created and is not in a stopped state, there is no need to kill it, as this may result in a false restart, such as killing a paused task.